{"uid":"2100731ae53cd530","name":"SyncManager sync throws error when documents sync fails","historyId":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager:.frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager#SyncManager sync throws error when documents sync fails","time":{"start":1743766847000,"stop":1743766847037,"duration":37},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false},"afterStages":[],"labels":[{"name":"resultFormat","value":"junit"},{"name":"suite","value":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager"},{"name":"testClass","value":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager"},{"name":"package","value":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":[]},"source":"2100731ae53cd530.json","parameterValues":[]}